ankidroid/Anki-Android
# Colpkg

## Purpose / Description
See #5107 and dae/anki@0c80b545

Note: during testing I also found two issues which are fixed in additional commits on this branch
1) crash opening note editor on API16
2) no way to save exported files to filesystem on API16

## Fixes
Fixes #5107

## Approach
Change the extension for collection export to ".colpkg", and allow these files to be imported

## How Has This Been Tested?
On API 28 and API 16 emulators:

* Export deck --> still goes to .apkg file
* Export collection --> now goes to .colpkg file
* import apkg deck above (via file manager and AnkiDroid menu) --> still imports in "add" mode
* import colpkg collection above (via file manager and AnkiDroid menu) --> imports in "replace" mode as expected
* Rename colpkg file above to "collection.apkg" and import --> imports in "replace" mode as expected
